packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -0,0 +1,135 @@
+import { Component, h, JSX, Element, Prop, State, Host } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import {
+  validateFormIds,
+  getSlottedInput
+} from '../../../stable/gux-form-field/gux-form-field.service';
+import { GuxSelectorCardVariant } from './gux-selector-card.types';
+import { calculateInputDisabledState } from '@utils/dom/calculate-input-disabled-state';
+import { onInputDisabledStateChange } from '@utils/dom/on-input-disabled-state-change';
+import { preventBrowserValidationStyling } from '@utils/dom/prevent-browser-validation-styling';
+import { logWarn } from '@utils/error/log-error';
+import { setSlotAriaDescribedby } from '../../../stable/gux-form-field/gux-form-field.service';
+
+/**
+ * @slot icon - Required slot for icon
+ * @slot input - Required slot for input tag
+ * @slot label - Required slot for label tag
+ * @slot content - Optional slot for additional text content
+ * @slot badge - Optional slot for badge
+ */
+
+@Component({
+  styleUrl: 'gux-selector-card.scss',
+  tag: 'gux-selector-card-beta',
+  shadow: true
+})
+export class GuxSelectorCard {
+  private input: HTMLInputElement;
+  private disabledObserver: MutationObserver;
+
+  @Element()
+  private root: HTMLElement;
+
+  //TODO: COMUI-2784 Descriptive variant styling
+  @Prop()
+  variant: GuxSelectorCardVariant = 'simple';
+
+  @State()
+  private disabled: boolean;
+
+  @State()
+  private checked: boolean;
```

Comment:
Is this state needed? If its only used for the styling would `this.input.checked` work ?

Reply:
this didn't work for me.

Reply:
Discussed with gillian I dont think this will work actually because this.input.checked when changed does not cause a re-render so it wont work so with the checked state when it changes it causes a re-render.